lidar_lite_v3hp/src/lidar_litev3hp_worker.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#include "lidar_litev3hp_worker.h"
+
+double LidarLiteNodeWorker::trailerAngle(double sensor_distance,double sensor_inp1,double sensor_inp2)
```

Comment:
With math functions like this, it is usually a good idea to provide a high level description of the math either a equation or a comment describing the concept or reference.